### PR TITLE
add /v1 for okta issuer

### PR DESCRIPTION
**Pull request by lquanx**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by nirojshrestha019 — APPROVED** [review]

File: src/config.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ const getIssuerEndpoint = async (suffix?: string) => {
 
 const getPatientPickerEndpoint = async (suffix?: string) => {
```
Comment (lwarren01) [inline src/config.ts]: I am a little confused as to why this function is called below to set the auth and token URLs. Why aren't they calling `getIssuerEndpoint` instead? Where is the patientPickerEndpoint used?

File: src/config.ts [hunk 2]
```diff
@@ -69,7 +69,7 @@ const getIssuerEndpoint = async (suffix?: string) => {
...
     if (patientPickerEndpoint === defaultEndpoint) {
-        patientPickerEndpoint = await getIssuerEndpoint();
+        patientPickerEndpoint = `${await getIssuerEndpoint()}/v1`;
```
Comment (lwarren01) [inline src/config.ts]: This doesn't look like a valid URL to me. But I guess I need to see where it is being referenced.

**Review by lwarren01 — COMMENTED** [review]
I am a little confused regarding the use of getPatientPickerEndpoint and how it is being used in config.ts

File: src/config.ts (hunk 1 above)
Comment (lquanx) [reply src/config.ts]: I will let them call getIssuerEndpoint  instead

File: src/config.ts (hunk 2 above)
Comment (lquanx) [reply src/config.ts]: I think I can try to remove method getPatientPickerEndpoint

**Comment by lquanx** [comment]
yeah it is confusing. I will make it simple for now.

File: src/config.ts [hunk 3]
```diff
@@ -42,12 +42,6 @@ let issuerEndpoint =
     process.env.ISSUER_ENDPOINT === 'undefined'
         ? defaultEndpoint
         : process.env.ISSUER_ENDPOINT;
-let patientPickerEndpoint =
```
Comment (lquanx) [inline src/config.ts:45]: removed patientPickerEndpoint which is not used anywhere

**Pull request merged**